infra/docker/android-sdk/Dockerfile

```diff
@@ -82,8 +74,10 @@ RUN ${ANDROID_HOME}/cmdline-tools/${CMDTOOLS_VERSION}/bin/sdkmanager --sdk_root=
 
 # Env variable for android build (ndk, gradle)
 ENV NDK_DIR ${ANDROID_HOME}/ndk/${NDK_VERSION}
-ENV GRADLE_HOME /home/ubuntu/gradle
+ENV GRADLE_HOME /opt/android-tools/gradle
 ENV PATH ${PATH}:${GRADLE_HOME}/bin:${ANDROID_HOME}/cmdline-tools/${CMDTOOLS_VERSION}/bin:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools
 
 # setup adb server
 EXPOSE 5037
+
+WORKDIR /home/ubuntu
```

Comment:
Just for note:

As I was told, this line is not for CI agent, but for someone who uses this script.